src/users/utils.js

```diff
@@ -17,5 +17,12 @@ export const UsersUtils = {
     }
 
     return language.slice(0, 2).toLowerCase()
+  },
+
+  getProviderCode(providerName) {
+    //to snake case
+    return providerName && providerName.match(/[A-Z]{2,}(?=[A-Z][a-z]+[0-9]*|\b)|[A-Z]?[a-z]+[0-9]*|[A-Z]|[0-9]+/g)
```

Comment:
Why do we need this? Is this from ticket requirements?

Reply:
loginWithOauth2 reveives providerName as the first param, but we have to add providerCode to a url. In Console we used lodash._snakeCase for that convertion, but as we don't use such libraries in SDK, i've created my own one alter.

Reply:
Please cover this function by tests

Reply:
Done. Is it ok, or should I test getProviderCode method through mocking loginWithOauth ?

Reply:
no, the login methods accept `provideCode` instead of `providerName` and this code we provide on the Console when the provider is enabled, so need to remove the `getProviderCode` method